src/Symfony/Bundle/FrameworkBundle/Controller/ControllerTrait.php

```diff
@@ -115,6 +115,27 @@ protected function redirectToRoute(string $route, array $parameters = [], int $s
         return $this->redirect($this->generateUrl($route, $parameters), $status);
     }
 
+    /**
+     * Returns a RedirectResponse to last accessed url.
+     *
+     * @author Temuri Takalandze (ABGEO) <takalandzet@gmail.com>
+     *
+     * @final
+     */
+    protected function redirectToBack(): RedirectResponse
+    {
+        // Get current request.
+        $request = $this->container
+            ->get('request_stack')
+            ->getCurrentRequest();
+
+        // Get referer URL.
+        $referer = $request->headers->get('referer');
+
+        // Redirect to referer url or /.
+        return $this->redirect($referer ?? '/');
```

Comment:
Should the fallback be configurable ? I'm quite sure that `/` won't be the best fallback for most business cases.